LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License
 
-Copyright (c) 2018-2020 Axis Communications AB
+Copyright (c) 2018-2022 Axis Communications AB
```

Comment:
2023 ?

Reply:
You just volunteered for a new PR after this one 😉